src/image_modifications.cpp

```diff
@@ -761,28 +760,35 @@ REGISTER_MOD_PARSER(CROP_TRANSPARENCY, )
 	return std::make_unique<crop_transparency_modification>();
 }
 
+// TODO: a more general solution?
+bool in_range(int val, int min, int max)
+{
+	return min < val && val < max;
+}
+
```

Comment:
More general how? I don't quite understand why this is a TODO. Isn't it about as general as you can get? The only way to be more general would be to make it a template.

Reply:
Meant maybe make it a general util function.

Reply:
Although, based on the usage of it a little further down, this code is wrong. Both comparisons should be `<=` for its use-case in `~BW()` at least; as is, it's going to claim that 0 and 255 are out of range, which is wrong.

Reply:
Fixed.